Src/examples/cURL/smtp_mime/smtp_mime.ecf

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<system xmlns="http://www.eiffel.com/developers/xml/configuration-1-16-0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+ xsi:schemaLocation="http://www.eiffel.com/developers/xml/configuration-1-16-0 http://www.eiffel.com/developers/xml/configuration-1-16-0.xsd" name="smtp_mime">
+	<target name="smtp_mime">
+		<root cluster="smtp_mime" class="APPLICATION" feature="make"/>
+		<option warning="true" is_obsolete_routine_type="true" syntax="transitional">
```

Comment:
I guess you can remove the part ` is_obsolete_routine_type="true" syntax="transitional" `

Reply:
done